### ISSUE TITLE
[System.Runtime.Intrinsics] API suggestion: Introduce an intrinsic all bits set field to Vector intrinsic types

### ISSUE THREAD

**Issue by john-h-k** [dotnet/corefx#40504]
Currently we have `Vector{nnn}<T>.Zero` type which the JIT recognises and transforms into a `xorps` or similar. It would be useful to have a `.One` (possibly a different name, as that could suggest the value one in each element), which is the inverse, having all bits set. The JIT could ideally generate this to be
```cmpps <dest>, xmm0, xmm0, 0x0``` which is recognised as dependency breaking on modern intel architectures for minimal cost. An all-ones value is useful, and can be frequently used for masking and mixing elements alongside the all-zero value.

**Comment by GrabYourPitchforks** [comment]
@tannergooding Weren't you looking into this a while back? Am I misremembering?

**Comment by tannergooding** [comment]
Yes, and I agree this would be something useful to expose. However, `One` is not an appropriate name since that might imply the wrong thing.

Something that conveys `AllBitsSet` would be better, but I'm not sure I have a good suggestion.

**Comment by GrabYourPitchforks** [comment]
API review - `AllBitsSet` seems like a decent name given the scenario.

Suggestion: also rename the internal member `Vector<T>.AllOnes` at the same time so that if it does become public in the future we have consistent naming already.
